src/ctr_exit.c

```diff
@@ -83,9 +79,16 @@ gboolean check_child_processes_cb(gpointer user_data)
 	return G_SOURCE_REMOVE;
 }
 
-gboolean on_sigusr1_cb(gpointer user_data)
+gboolean on_signalfd_cb(gint fd, GIOCondition condition, gpointer user_data)
```

Comment:
`condition` is not used and the CI barks about it (just prepend G_GNUC_UNUSED)

Reply:
Funny.  I had caught this locally as well, and changed it, but forgot to commit before pushing.  Fixed now :)